docs/product/issues/issue-details/replay-issues/hydration-error.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Hydration Error
+sidebar_order: 50
+description: "Learn about hydration errors."
+---
+
+Hydration errors are a React-specific problem that happen when the initial client UI does not match what was rendered on the server. They can result in extra work for the browser, and a slower pageload experience for users.
+
+## Prerequisites for Seeing Hydration Error Diff Tools
+
+Hydration Errors can come into your issue stream either as regular error events, or they could be detected inside a replay session and converted into an issue by the server. Read more about
+[configuring hydration errors](/platforms/javascript/session-replay/issue-types/) to learn about the differences.
+
+If the hydration error event instance has an associated replay session, then you will see diff tools on the issue details page to make debugging the hydration error easier.
```

Comment:
```suggestion
If the hydration error event instance has an associated replay, then you will see diff tools on the **Issue Details** page to make debugging the hydration error easier.
```

Reply:
isn't this always the case since these issues are session replay-based? i think this paragraph/section needs to be re-thought.

we should start from the standpoint of 'session replay specific hydration errors' and talk about what those entail.

then, in a small callout box we can reference the traditional hydration error issues and how those are different and filtered out by default etc

Reply:
both replay-hydration and error-hydration get the same set of diff-tools if a replay is attached. It just turns out that in the former case a replay is guaranteed to be there. How the issue got into the stream is a bit of an implementation problem though, and probably we should only talk about it in terms of the preferences (which people shouldn't need to touch imo).

I don't think we should talk too much about "traditional hydration error" being very different, because when you're debugging the issue it's not really different, the page looks the same.

Reply:
@ryan953  🫨 oooh i didn't know that replay-hydration AND error-hydration both get the new and improved diffing treatment.

i still find referencing both kind of issues here confusing and its not clear what each one entails/why it matters

i think it makes sense to focus on replay-hydration since that's this page is dedicated on, and the other traditional hydration error is more an 'aside'/callout

Reply:
maybe the page needs to be just about hydration errors in general, and the inbound-filters and replay-project-settings stuff can be somewhere less prominent (therefore less confusing)